pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/index.md

```diff
@@ -0,0 +1,325 @@
+
+- [**Phasor $$I_1$$**](./PhasorI1/): The dynamics of the current loop are approximated by a first-order transfer function with a given time constant $$\tau_c$$. 
+
+- [**Phasor $$I_0$$**](./PhasorIo/): The current loop is completely removed (as it has the fastest dynamics of the system), which requires a reformulation of the power loop. The output of the later will be directly considered as the converter current.  
+
+- [**Phasor $$PQ_1$$**](./PhasorPQ1/): The dynamics of the power loop are approximated by a first-order transfer function with a given time constant $$\tau_{pq}$$, similarly to the first case. The current loop is also removed.
+
+The following figure (which corresponds to Figure 6 of [[1]](#1)) depicts the performance for all these models compared to the EMT model in one particular scenario, in which the power setpoint tracking for $$P^*, Q^*$$ was studied:
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; padding:4px 0;">
+<img src="{{ '/pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/ErrorPlot_Phasor.svg' | relative_url }}"
+     alt="Phasor Models Comparison"
+     style="float: center; margin-right: 10px; width: 700px;" />
+</div>
+<div align = 'center'>
+Figure 8: Phasor Models Error Comparison <a href="#1">[1]</a>
+</div>
+<br>
+
+As a summary, the Full-Phasor model is proven as a reliable although slower model. It is capable to solve with good accuracy the system for time step sizes that are outside the convergence range of the EMT model. The $$I_0$$ and $$I_1$$ Phasor models are able to reach convergence at bigger time steps with a decent accuracy (around $$10^3$$ $$\mu s$$), while the $$PQ_1$$ model is the fastest but with a good accuracy compared to the rest of the models for bigger time steps. Generally, choosing between one or other model depends on the time step size desired to use, rather than the type of study, as all the phasor models can work in a similar range of frequencies, as seen in the following figure:
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; padding:4px 0;">
+<img src="{{ '/pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/Phasor_suitability.svg' | relative_url }}"
+     alt="Phasor Models Suitability"
+     style="float: center; margin-right: 10px; width: 500px;" />
+</div>
+<div align = 'center'>
+Figure 9: Phasor Models Suitability <a href="#1">[1]</a>
+</div>
+<br>
+
+The details can be consulted in:
+
+{% include subfolderNav.html %}
+
+
+## Open source implementations
+
+This model has been successfully implemented in :
+
+| Software      | URL | Language | Open-Source License | Last consulted date | Comments |
+| --------------| --- | --------- | ------------------- |------------------- | -------- |
+<!-- Yet to be reviewed -->
+
```

Comment:
Give proper format and comments if not filled.

Reply:
No implementations found, removed